src/commands/general/setParser.ts

```diff
@@ -0,0 +1,51 @@
+import { PromiseSlashCommandBuilder, command } from "../../utils";
+import { allParsers } from "../../parser";
+import { prisma } from "../../utils/prisma";
+
+const list = allParsers.map((parser) => {
+    return {
+        name: parser.name,
+        value: parser.name,
+    };
+})
+
+const meta = PromiseSlashCommandBuilder(
+	"setparser",
+	"Sets the parser for a specific channel"
+).then((builder) => {
+    return builder.addStringOption((option) =>
+        option
+            .setName("parser")
+            .setDescription("The parser to use")
+            .setRequired(true)
+            .addChoices(list)
+    )
+})
+
+export default command(meta, 
+    async ({ interaction }) => {
+        await interaction.deferReply({ ephemeral: true });
+        const parser = interaction.options.getString("parser");
+        const channelid = interaction.channel?.id;
+        const guildid = interaction.guild?.id;
+
+        if (!channelid || !guildid) return interaction.editReply("Not in a guild channel");
+        if (!parser) return interaction.editReply("No parser provided");
+
+        const parserDB = await prisma.parserdle.upsert({
```

Comment:
please check if the selected parser is already the set parser, if so print a warning message.

If possible give the user the ability to change it in that message.

Reply:
Of course i can do it, but it would take longer. i would build it similar to the help comand.

Reply:
I have changed it so that it looks like this:

![image](https://github.com/user-attachments/assets/0aaf4bbd-6850-4170-926f-67f44df05d2a)